src/index.html

```diff
@@ -13,5 +13,164 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal Informations</legend>
+        <div class="question">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
        <legend>Personal Informations</legend>
        
        <div class="question">
```